.github/workflows/cd-syftcli.yml

```diff
@@ -0,0 +1,102 @@
+name: CD - syft-cli
+
+on:
+  schedule:
+    - cron: "00 10 * * */3" # At 10:00 UTC on every three days
```

Comment:
MAYBE: We could it release on every seven days, since we would be also releasing github releases,it would create a lot tags (as we also have a provision to manually release )